src/rust/madara/sources/hiperdex/res/source.json

```diff
@@ -0,0 +1,18 @@
+{
+    "info": {
+        "id": "en.hiperdex",
+        "lang": "en",
+        "name": "Hiperdex",
+        "version": 1,
+        "url": "https://1sthiperdex.com",
+        "nsfw": 2
+    },
+    "listings": [
+        {
+        "name": "Trending"
+        },
+        {
+        "name": "Popular"
+        }
```

Comment:
make sure the json file is formatted and using tabs `jq --tab . source.json | sponge source.json`